astTraversal/testfiles/callExpression.go

```diff
@@ -45,3 +45,48 @@ func callExpr() {
 func TestFunction(a int, b string) string {
 	return fmt.Sprintf("%d - %s", a, b)
 }
+
+// nolint:unused
```

Comment:
It might be an idea to flag these files as unused in the `.golangci.yml` if you you change line 66 to this `path-except: 'test\.go\\astTraversal/testfiles\.go'`, it should exclude those files from unused lint errors.

Reply:
Modified